contracts/src/interfaces/AbstractStakeTable.sol

```diff
@@ -81,14 +81,14 @@ abstract contract AbstractStakeTable {
 
     // === Queuing Stats ===
 
-    /// @notice Get the next available epoch for new registration
-    function nextRegistrationEpoch() external view virtual returns (uint64);
+    /// @notice Get the next available epoch and queue size after new registration
+    function nextRegistrationEpoch() external view virtual returns (uint64, uint64);
     /// @notice Get the number of pending registration requests in the waiting queue
     function numPendingRegistrations() external view virtual returns (uint64);
-    /// @notice Get the next available epoch for exit
-    function nextExitEpoch() external view virtual returns (uint64);
+    /// @notice Get the next available epoch for exit and queue size after new exit request
+    function nextExitEpoch() external view virtual returns (uint64, uint64);
```

Comment:
not "queue size" but "queue size *in that epoch*",

by default, ppl would assume it's overall queue size.

Reply:
Yes, done in e0c43fd.